src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
 // Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <div>
+    {todos.map(todo => (
+      <section className="TodoList">
```

Comment:
Not fixed. For now, each item is a separate section. But you have to create one list with different items 
![image](https://github.com/mate-academy/react_static-list-of-todos/assets/99403198/388be0a0-6eef-4a7e-9401-d5f0a7690378)
```suggestion
  <section className="TodoList">
    {todos.map(todo => (
```